Formula/chapel.rb

```diff
@@ -12,23 +12,36 @@ class Chapel < Formula
     sha256 x86_64_linux: "ca34ea32e25b7c9fea13bf25a09ac8b3151f45f8b8ac2dfeebfc0d18773783ba"
   end
 
+  depends_on "llvm@11"
   depends_on "python@3.9"
 
   def install
     libexec.install Dir["*"]
     # Chapel uses this ENV to work out where to install.
     ENV["CHPL_HOME"] = libexec
     # This is for mason
-    ENV["CHPL_REGEXP"] = "re2"
+    ENV["CHPL_RE2"] = "bundled"
```

Comment:
Can't it use the system/homebrew one?

Reply:
@SMillerDev:  No, as we discussed [here](https://github.com/chapel-lang/chapel/issues/17428#issue-835130875), we have patches to re2 to make it work for us that have not been accepted upstream.